docs/_articles/en/v3.x/scanner-commands/run.md

```diff
@@ -149,3 +149,29 @@ $ sfdx scanner:run --target "/some-project/" --severity-threshold 2
 
 ## Demo
 ![Run Example](./assets/images/run.gif)
+
+## Target Patterns
```

Comment:
This feels like it should be part of the `Additional Notes` section above.

Reply:
Good idea

Reply:
I moved the info to this section. How does it look now? I'm not sure how to get rid of the space between the main bullet point and the nested bullet point.

Reply:
Yeah, I'm not loving that gap either.
Maybe we could put it in the FAQ or Troubleshooting?

Reply:
There are already a couple questions in the FAQ that mention the target patterns, so I think adding it there would work. Do you think "Questions about language support" would be a good section?

Reply:
I'd say it's probably best to have it in Troubleshooting. Let's get another opinion from the team though.

Reply:
@jfeingold35 @gkjung Wondering if we need to bring this more visibility than in the Troubleshooting section. What do you think of here: https://forcedotcom.github.io/sfdx-scanner/en/v3.x/architecture/eslint-engine/#how-does-salesforce-code-analyzer-use-eslint - since this config change would be particular to how users can get the best out of eslint and eslint-lwc through SFCA.

Reply:
I think ESLint would be a good spot for it. After Teresa's feedback, this is what the message to the user looks like `A file was processed by both ESLint and ESLint-LWC simultaneously, which could result in duplicate violations. To customize which files each engine targets, check out our documentation on target patterns.` Could we make it "`check out our documentation under ESLint about target patterns` or something a little more descriptive so they know where to look in the documentation? She said not to add a link but there are so many pages and no search bar that I think it would take some digging no matter where it is unless we add a descriptor

Reply:
Ideally, information about `targetPattern` should be under a section about our Config.json file. But we don't have it at the moment and I don't think we should expand the scope of this story anymore. I just added a suggestion to the conversation thread in your other PR. Let me know your thoughts.

Reply:
I'm starting to lean towards including this information in Troubleshooting/FAQ as well. We could put the same message that the users would see and talk about why it shows up and how they can address it.